asmcli/lib/checks.sh

```diff
@@ -464,12 +464,14 @@ istio_namespace_exists() {
 }
 
 is_autopilot() {
-  # TODO: wait till https://pkg.go.dev/google.golang.org/genproto/googleapis/container/v1#Cluster  
-  # publish the `Autopilot` field
-  # This is a temporary workaround to check if a cluster is Autopilot or GKE
-  # This CRD will be installed only if the cluster is Autopilot
   if [[ "${IS_AUTOPILOT}" -eq 1 ]]; then return; fi
-  if ! retry 2 kubectl get crd allowlistedworkloads.auto.gke.io 1>/dev/null 2>/dev/null; then
+
+  local _IS_AUTOPILOT; _IS_AUTOPILOT="$(gcloud container clusters describe "${CLUSTER_NAME}" \
```

Comment:
_IS_AUTOPILOT implies a boolean to me. Can we somehow make this a 0 or 1?

Reply:
updated var name to match GCP field name